### DISCUSSION TITLE
Modding Feature Request: Luminosity(brightness)

### DISCUSSION THREAD

**Discussion by G-M-twostay** [FakeFishGames/Barotrauma#13313] [Code questions and modding support]
I'd like a way to check for lights and luminosity. I don't think there is a way to do this through pure XML currently. This way we can make light sensors or items/monsters that reacts to light.

It would also be helpful if anyone can point out a way of how this is done in code currently.

**Comment by jigardi** [comment]
Sounds interesting, have my upvote!
